util/build_configs/setenv-example-1.bash

```diff
@@ -53,11 +55,26 @@ if [ -z "$BUILD_CONFIGS_CALLBACK" ]; then
 
     substrates=smp,udp      # two widely-supported gasnet substrates for this example
 
+
     # Use build_configs.py to make Chapel with and without gasnet
 
     ./build_configs.py -p $dry_run $verbose -s setenv-example-1.bash -l setenv-example-1.make.log \
         --comm=none,gasnet --substrate=none,$substrates  # no make target
 
+        # The following matrix of six Chapel runtime configurations are defined by the
+        # "--comm" and "--substrate" arguments passed to build_configs.py, above.
+        #
+        #                           CHPL_COMM=none    CHPL_COMM=gasnet
+        #                           ----------------  ----------------
+        # CHPL_COMM_SUBSTRATE=none         Make
```

Comment:
It may be easier to follow if we're consistent with how we display the build matrix. I noticed that example 3 differs from 1 and 2.

Reply:
They are very different "matrixes". 

1) Example 1/2 has holes- ie, Chapel make is skipped for build configs that don't make sense in Chapel. 
   Example-3 has no holes. Every possible build config is actually built. 

2) Example 1/2 is 2-D. Example 3 is 3-D.

I don't know how to show the example 3 build matrix in the same style as example 1/2.  

It could go the other way, and show example 1/2 matrix in the same style as example 3, like this:
```
        # CHPL_COMM   CHPL_COMM_SUBSTRATE   Make Chapel?
        # ---------   -------------------   ----------------
        # none        none                  Yes
        # none        mpi                       (skip)
        # none        udp                       (skip)
        # gasnet      none                      (skip)
        # gasnet      mpi                   Yes
        # gasnet      udp                   Yes
```
This doesn't seem like an improvement.

Reply:
I see. It's probably fine to leave as is then.